### PR TITLE
[ENH] Process text

### PR DESCRIPTION
**Pull request by samukweku**
# PR Description

Please describe the changes proposed in the pull request:

- Modifies only existing column
- Deprecate `new_column_names` and `merge_frame`
- keeping it simple (and easier to maintain)

**This PR improves `process_text`.**

# PR Checklist

Please ensure that you have done the following:

1. [x] PR in from a fork off your branch. Do not PR from `<your_username>`:`dev`, but rather from `<your_username>`:`<feature-branch_name>`.

2. [x] If you're not on the contributors list, add yourself to `AUTHORS.rst`.

3. [x] Add a line to `CHANGELOG.md` under the latest version header (i.e. the one that is "on deck") describing the contribution.
    - Do use some discretion here; if there are multiple PRs that are related, keep them in a single line.

# Automatic checks

There will be automatic checks run on the PR. These include:

- Building a preview of the docs on Netlify
- Automatically linting the code
- Making sure the code is documented
- Making sure that all tests are passed
- Making sure that code coverage doesn't go down.

# Relevant Reviewers

Please tag maintainers to review.

- @ericmjl

### CODE REVIEW

**Comment by ericmjl** [comment]
I think I know what's going on with the code coverage - when `dev` has more lines of code than PR branch, it'll detect a decrease in the code coverage. Probably continually merging `dev` into the PR branch will fix that problem.

**Comment by samukweku** [comment]
@nvamsikrishna05 @loganthomas @ericmjl can I bother you a tiny bit to have a look at this PR. thanks

**Review by nvamsikrishna05 — APPROVED** [review]
Looks good. 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@
 -   [EHN] Set `expand_column`'s `sep` default is `"|"`, same to `pandas.Series.str.get_dummies`. Issue #876. @Zeroto521
 -   [ENH] Deprecate `limit` from fill_direction. fill_direction now uses kwargs. @samukweku
 -   [ENH] Added `conditional_join` function that supports joins on non-equi operators. @samukweku
+-   [ENH] Deprecate `new_column_names` and `merge_frame` from `process_text`. Only existing columns are supported. @samukweku
```
Comment (ericmjl) [inline CHANGELOG.md]: Let's take this line up to the top.

**Review by ericmjl — APPROVED** [review]
Only one minor change requested, @samukweku :smile:.

When it's done, @nvamsikrishna05 please do the honors!

**Comment by samukweku** [comment]
@nvamsikrishna05, my apologies; can i bother u with this PR? A second look and possible merge if all is good

**Review by nvamsikrishna05 — APPROVED** [review]
@samukweku Looks good. Sorry, I missed this PR for so long. Go ahead and merge it once all checks pass.

**Pull request merged**